addons/common/backwards_comp.sqf

```diff
@@ -71,3 +71,11 @@ CBA_fnc_intToString = {
     if (isNil "_int") exitWith {""};
     str _int
 };
+
+CBA_fnc_inArea = {
```

Comment:
That's a general problem with this file and should probably be fixed in another PR